src/pages/Help/ZHelp.css

```diff
@@ -0,0 +1,40 @@
+.flex_container {
+  display: flex;
+  flex-direction: column;
+
+  align-items: center;
+  margin: 75px;
+  padding: 10px;
+}
+
+ol.flex_list {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 2vw;
+  font-size: 1.5rem;
+  background: #efa72d;
```

Comment:
Wow yellow background is too much ^^ would be easier to read with white background ;)

I like the hover and the icon, that's great job 👍

Reply:
opacity everywhere